iceoryx_hoofs/include/iceoryx_hoofs/internal/units/duration.hpp

```diff
@@ -38,7 +38,7 @@ enum class TimeSpecReference
 
 class Duration;
 
-inline namespace duration_literals
+namespace duration_literals
```

Comment:
Please revert this change. The `inline namespace` is needed to make the symbols visible in the outer namespace, see [here](https://stackoverflow.com/a/11018418) for more infos.

Reply:
I know what it does but that is something which is not public API and is something which can become dangerous because we implicitly activate this namespace as well.
In general I think we should not use `inline namespaces` at all.

Just think of the problems we encounter when we define other literals as well which are also named `m` like meter for instance and put them into such a inline namespace as well - with have ambiguous symbols.